lib/node/runner/api/account.go

```diff
@@ -52,3 +56,243 @@ func (api NetworkHandlerAPI) GetAccountHandler(w http.ResponseWriter, r *http.Re
 
 	httputils.MustWriteJSON(w, 200, payload)
 }
+
+func (api NetworkHandlerAPI) GetFrozenAccountsByAccountHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	address := vars["id"]
+	options, err := storage.NewDefaultListOptionsFromQuery(r.URL.Query())
+	if err != nil {
+		http.Error(w, errors.ErrorInvalidQueryString.Error(), http.StatusBadRequest)
```

Comment:
```suggestion
		httputils.WriteJSONError(w,err)
```
If this `err` is suitable for rendering.